app/main.py

```diff
@@ -1,2 +1,71 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.length = 0
+        self.hash_table: list = [None] * 8
+
+    def __getitem__(
+            self, key: int | float | bool | str | tuple
```

Comment:
it's better to use Hashable typehint for the "key"

Reply:
Done